presto-main/src/main/java/com/facebook/presto/cost/HistoryBasedPlanStatisticsTracker.java

```diff
@@ -248,8 +248,9 @@ private PartialAggregationStatistics constructAggregationNodeStatistics(PlanNode
         PlanNodeStats childNodeStats = planNodeStatsMap.get(childNode.getId());
         if (childNodeStats != null) {
             double partialAggregationInputBytes = adjustedOutputBytes(childNode, childNodeStats);
-            return new PartialAggregationStatistics(Estimate.of(partialAggregationInputBytes),
-                    Estimate.of(outputBytes),
+            return new PartialAggregationStatistics(
```

Comment:
There is a method `Estimate.estimateFromDouble` which does this. Naming is terrible as we have methods like `Estimate.of` as well. We can change these names to be more explicit of what they do and use right methods.